static/scss/homepage.scss

```diff
@@ -221,7 +221,9 @@ body.app-media {
       }
 
       @include breakpoint(phone) {
-        display: none;
+        font-size: 20px;
+        line-height: 1.15;
```

Comment:
@rhysyngsun [According to MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height), you _can_ provide a unit, [but it's better not to](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#Prefer_unitless_numbers_for_line-height_values).